examples/nlp/gpt/conf/gpt_ppo_critic.yaml

```
@@ -19,6 +17,29 @@ trainer:
     max_steps: 10000
     gradient_clip_val: 1.0
 
+    # a PyTriton parameter to specify
+    # the max time a request will be delayed in the scheduling queue
+    # to wait for additional requests for batching
+    # for more info see: https://github.com/triton-inference-server/pytriton/blob/9e789db73880ecc667845455bbd2c19601e6279b/pytriton/model_config/common.py#L76
+    max_queue_delay_microseconds: 2000
+
+    # strips the padded sequence length from requests that come in
+    # when requests are sent with token, sequence length format
+    # we will look at the sequence length and see if we can strip
+    # the request to a lower sequence length that is a multiple of
+    # the specified below
+    strip_sequence_length_to_multiple: null
+
+    # this parameter tells PyTriton what our preferred batch sizes are
+    # for request batching. This parameter is distinct from
+    # the model.forward_micro_batch_size to optimize for cases where Pipeline Parallelism
+    # is enabled. Because in that case we will want PyTriton to give us
+    # a very large batch, but then in the model we will cut it into smaller
+    # batches(determined by model.forward_micro_batch_size) to fill the pipe better.
```

Just my usual nit on blank spaces before parentheses :p
```suggestion
    # batches (determined by model.forward_micro_batch_size) to fill the pipe better.
```